TransformationSystem/Client/test/Test_Client_TransformationSystem.py

```diff
@@ -12,6 +12,21 @@
 
 #############################################################################
 
+
+class reqValFake_C(object):
```

Comment:
Because of this (we run without configuration, remember):

======================================================================
ERROR: test_prepareTranformationTasks (__main__.RequestTasksSuccess)
----------------------------------------------------------------------
Traceback (most recent call last):
   File 
"TransformationSystem/Client/test/Test_Client_TransformationSystem.py", 
line 49, in setUp
     requestClient = self.mockReqClient,
   File 
"/home/toffo/pyDevs/DIRAC/TransformationSystem/Client/TaskManager.py", 
line 114, in __init__
     self.requestValidator = RequestValidator()
   File "/home/toffo/pyDevs/DIRAC/Core/Utilities/DIRACSingleton.py", 
line 61, in __call__
     cls.instance = super( DIRACSingleton, cls ).__call__( *args, **kwargs )
   File 
"/home/toffo/pyDevs/DIRAC/RequestManagementSystem/private/RequestValidator.py", 
line 103, in __init__
     configPath = PathFinder.getAgentSection( 
"RequestManagement/RequestExecutingAgent" )
   File 
"/home/toffo/pyDevs/DIRAC/ConfigurationSystem/Client/PathFinder.py", 
line 42, in getAgentSection
     systemSection = getSystemSection( agentName, agentTuple, setup = 
setup )
   File 
"/home/toffo/pyDevs/DIRAC/ConfigurationSystem/Client/PathFinder.py", 
line 30, in getSystemSection
     instance = getSystemInstance( serviceTuple[0], setup = setup )
   File 
"/home/toffo/pyDevs/DIRAC/ConfigurationSystem/Client/PathFinder.py", 
line 24, in getSystemInstance
     raise RuntimeError( "Option %s is not defined" % optionPath )
RuntimeError: Option /DIRAC/Setups/None/RequestManagement is not defined